src/System.Console/src/System.Console.builds

```diff
@@ -9,23 +9,8 @@
       <OSGroup>Windows_NT</OSGroup>
     </Project>
     <Project Include="System.Console.csproj">
-      <OSGroup>Unix</OSGroup>
-      <TargetGroup>netstandard1.3</TargetGroup>
-    </Project>
-    <Project Include="System.Console.csproj">
-      <OSGroup>Windows_NT</OSGroup>
-      <TargetGroup>netstandard1.3</TargetGroup>
-    </Project>
-    <Project Include="System.Console.csproj">
-      <TargetGroup>net46</TargetGroup>
-    </Project>
-    <Project Include="System.Console.csproj">
       <TargetGroup>net463</TargetGroup>
     </Project>
-    <Project Include="System.Console.csproj">
```

Comment:
We need to add back this netcore50 build. By removing it you ended up throwing out some changes we needed which we started hitting in our xunit runner for UWP recently https://github.com/dotnet/buildtools/issues/1172.